paper.tex

```diff
@@ -1109,84 +1109,110 @@ \section*{ARG simplification}
+(C) Remove nodes that are unary in all local trees. This removes both recombinant nodes
+that have only one child (e.g. \textsf{N}) and nodes that have more
+than one child (i.e. are ``common ancestors'') but never represent coalescences
+in the local trees (e.g. \textsf{R}).
+(D) Rewrite edges to bypass any unary nodes in local trees. Note that this loses lineage
+information such as the fact that \textsf{K} is connected to \textsf{A} via \textsf{E}; it
+can also create many extra roots.
 }
 \end{figure}
 
-% Q: would it help to talk about ``tree vertice'' here or would it
+% Q: would it help to talk about ``tree vertices'' here or would it
 % just be more confusing? I worry that people would think that
 % they are two different things then, and it's really important
 % that they get that a node-is-a-node
+
+
 The ideas of ``meaningful'' local tree topologies revolve around the
 presence of ``unary'' nodes: a node in the tree that has exactly
-one child. For example, Fig.~\ref{fig-simplification}E shows the
-local trees along the genome for a simulated ARG
-(Fig.~\ref{fig-simplification}A). Consider the right-most tree,
-and the path above node $D$. Before $D$ meets its common ancestor
-over the interval $(93, 100]$ with the other samples it passes through
-nodes $H, K, N, R$ and $S$, reflecting the corresponding
-traversal through the ARG (see section XXX). Such unary nodes
-in a local tree are not usually considered significant,
+one child. Fig.~\ref{fig-simplification} show a series of successive steps of
+simplification, starting with a complete gARG simulated under a Wright-Fisher
+model, shown both as a mathematical graph on the left, and the equivalent set of 6
+local trees on the right (Fig.~\ref{fig-simplification}A). Consider the path above node
+\textsf{A} in the penultimate tree, highlighted in yellow and covering genomic positions
+60 to 70. On this path, before $A$ meets its common ancestor with the other samples at
+node \textsf{Q}, it passes through  \textsf{E},  \textsf{J},  \textsf{M},  \textsf{N} and 
+\textsf{P}, which in this region of the genome are all unary nodes. This reflects the
+passage of this span of genome upwards through the ARG (see section XXX).
+Such unary nodes in a local tree are not usually considered important,
 because they provide very little information and are not
 detectable by information from the tree. % weak
-The nodes in a tree usually mark the points where samples find a
-\emph{common} ancestor, rather than focusing on specific
-ancestors. For most purposes, we would consider an edge
-directly joining sample $D$ to the node $T$ to be more
-useful and informative. Simplification is the process of removing
-these uninformative and (for most computational purposes)
-inefficient unary paths.
+In the local trees, attention is usually focussed upon points where upwards
+lineages meet in a \emph{common} ancestor (the MRCA node, or coalescence point).
```

Comment:
I think the original first sentence is better here. The "coalescence point" terminology doesn't do much for me, and I don't see what talking about upwards lineages helps.

Reply:
OK, but I think "uninformative" is a little strong. I put in coalescence, as that's what most popgen people would use to describe such a node in the local trees, so thought it would aid familiarity. Similarly with MRCA. How about:

> The nodes in a tree usually mark the points where samples find a \emph{common} ancestor, such as node $Q$ in the highlighted tree. For many purposes, we would therefore consider a path directly joining sample $A$ to node $Q$ to be of equivalent status to the path containing the intermediate unary nodes. Simplification is the process of removing these less informative and (for most computational purposes) inefficient nodes from the local trees and the corresponding annotated graph.

I think we need to talk about removing nodes from the trees because the last step (c->d) doesn't remove graph nodes.

Reply:
Very minor comment which may not end up being relevant here depending on how things are drafted, but I'm always a bit hesitant to call things "upward" or "downward". We're very consistent with drawing all of our trees in the same direction, but there is nothing close to a universal convention in the literature. I'd prefer "leafward" and "rootward", or words to that effect.

Reply:
Yes, ISWYM. It was to avoid "backward" vs "forward", because I think some people instinctively think backwards anyway, so using that adjective can be a bit jarring.

Worth bearing in mind in general, I agree. Thanks